### PR TITLE
Remove local swagger.json files and point to remote

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-chat/swagger/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ To update generated files for chat service, run the following command
 
 ### Code generation settings
 ``` yaml
-input-file: swagger.json
```
Comment (DominikMe) [inline sdk/communication/azure-communication-chat/swagger/README.md:37]: Let's remove the instruction on top of the README to run the swagger_replace.ps1 script

File: sdk/communication/azure-communication-chat/swagger/README.md [hunk 2]
```diff
@@ -51,3 +51,97 @@ context-client-method-parameter: true
 enable-xml: false
 required-parameter-client-methods: true
 ```
+### Rename AddChatThreadMembersRequest to AddChatThreadMembersOptions
```
Comment (DominikMe) [inline sdk/communication/azure-communication-chat/swagger/README.md:54]: Good job figuring that out!

(I'm assuming you validated by regenerating 🙃)

**Review by DominikMe — APPROVED** [review]
Awesome!

One comment about updating the swagger generation instructions.

File: sdk/communication/azure-communication-chat/swagger/README.md (hunk 2 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-chat/swagger/README.md:54]: haha thanks, and yes I have regenerated and validated that files are renamed correctly!

**Pull request merged**